pom.xml

```diff
@@ -279,6 +278,19 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>jbpm-installer-exclude-prod-productized-false</id>
```

Comment:
I would rename this to something like `prod-modules` or `prod-profile`

Reply:
sure, i just over took what was in your original change

Reply:
> This collides with `fullProfile`
> 
> https://github.com/kiegroup/jbpm/blob/c5c9793255ebb8dd2a9f52968f9bd7494e878a91/pom.xml#L201
> 
the full profile is generally used in all repositories where there is a need to differentiate something like basic + full (checks and distribution modules/release)
see https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L1791-L1797
so as you pointed below the release build use that too, so we need to keep or change a little bit the behavior for combined launching with full and not productized profile

> For your information, jbpm-distribution is referenced by community (except jbpm) or production projects like this
> 
> ```
> ./droolsjbpm-tools/drools-eclipse/org.jbpm.eclipse/plugin.xml:               url="http://repository.jboss.org/nexus/content/groups/public-jboss/org/jbpm/jbpm-distribution/6.2.0.Final/jbpm-distribution-6.2.0.Final-bin.zip"
> ./droolsjbpm-build-bootstrap/script/release/prepareUploadDir.sh:cp $deployDir/org/jbpm/jbpm-distribution/$kieVersion/jbpm-distribution-$kieVersion-bin.zip jbpm-$kieVersion-bin.zip
> ./droolsjbpm-build-bootstrap/script/release/prepareUploadDir.sh:cp $deployDir/org/jbpm/jbpm-distribution/$kieVersion/jbpm-distribution-$kieVersion-examples.zip jbpm-$kieVersion-examples.zip
> ```
> 
> ```
> ./rhba/rhpam/target/jbpm-installer-7.33.0-SNAPSHOT/build.properties:#   jBPM.url=https://repository.jboss.org/nexus/content/groups/public-jboss/org/jbpm/jbpm-distribution/${jBPM.version}/jbpm-distribution-${jBPM.version}-bin.zip
> ```
> 
> I think we can delete the fullPorfile since we build it with `productized` profile and the ones from community are not relevant. WDYT?
> Anyway it's much simpler solution than mine #1630